README.md

```diff
@@ -1,4 +1,26 @@
-# Gasket
+<p align="center">
+  <img alt="Gasket" src="site/logo-cover.svg" width="496">
+</p>
+
+<p align="center">
+Framework Maker for JavaScript Applications
+</p>
+
+---
+<div align="center">
+
+[![License][license-badge]][license]
+
+[![Contributors][contributors-badge]][contributors]
+[![PRs Welcome][prs-welcome-badge]][prs-welcome]
+[![Code of Conduct][coc-badge]][coc]
+
+<!-- [![install size](https://packagephobia.com/badge?p=@gasket/cli)](https://packagephobia.com/result?p=@gasket/cli)
+[![downloads](https://img.shields.io/npm/dm/@gasket/cli)](https://npmcharts.com/compare/@gasket/cli?minimal=true) -->
```

Comment:
Since `@gasket/cli` doesn't effect browser bundles I wouldn't have a this badge. Were you considering download still? If not, go ahead and remove it.

Reply:
Yeah, just had these here as a reference for if we wanted to add them to other package README's. Will remove them.